public/assets/webworker.js

```diff
@@ -26,7 +26,7 @@ self.addEventListener(
     if (oEvent.data.type == 'r') {
       //continue running
       readTest.push('\n');
-      elfRunNextInst();
+      elfRunNextInst(null);
```

Comment:
maybe a comment in the header of `elfRunNextInst` describing the behavior when command is null or non-null. would be useful for the reader